modules/ArchiveLibrary/File/Observers/FileObserver.php

```
@@ -124,13 +127,9 @@ public function created(File $file): void
 
```

The empty catch block for `\Exception` was removed but no alternative error handling was added. Consider either:
- Re-adding error logging
- Or explicitly documenting why exceptions can be safely ignored here